libs/blocks/marquee/marquee.css

```diff
@@ -326,13 +326,27 @@
     max-width: calc(50% - var(--grid-column-width));
   }
 
-  .marquee.split .media.bleed {
+  html:not([dir="rtl"]) .marquee.split .media.bleed {
```

Comment:
Adding `html:not([dir="rtl"])` will just add unneeded specificity to your code. Most of the code is duplicated, I think it can be improved like so:
```css
.marquee.split .media.bleed {
  position: absolute;
  width: 50vw;
  height: 100%;
  right: 0;
}

html[dir="rtl"] .marquee.split .media.bleed {
  // will inherit all declarations from .marquee.split .media.bleed
  left: 0;
  right: auto;
}
```

This way you inherit all default properties for the element and only override what you need to support your use-case